sklearn/gaussian_process/kernels.py

```diff
@@ -387,7 +388,7 @@ class StationaryKernelMixin(object):
     """
 
     def is_stationary(self):
-        """Returns whether the kernel is stationary. """
+        """Returns whether the kernel is stationary."""
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.